includes/classes/class-omise-card-image.php

```diff
@@ -12,7 +12,7 @@ class Omise_Card_Image {
 		 */
 		private static function get_image( $file, $alternate_text ) {
 			$url = WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' );
-			return "<img src='$url/$file' width='38px' alt='$alternate_text' />";
+			return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```

Comment:
@nimid sry, not sure I'm on the same page as you 😅 

From what I think is, right now you have
```php
return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```

just add another attribute, `class='OmiseCardBrandIcons'` there

So, it's gonna be,
```php
return "<img src='$url/$file' class='OmiseCardBrandIcons' style='width: 38px;' alt='$alternate_text' />";
```

and whenever (maybe rare case) customer want to override it, they can do
```css
.OmiseCardBrandIcons {
  width: 80px !important;
}
```

`!important` may not be the good choice, but at least, it allows customer to override `inline stylesheet` in case they want to, without touch our plugin files.